code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1915,20 +1915,6 @@ Eyes need to have significantly high darksight to shine unless the mob has the X
 
 	..()
 
-
-/mob/living/carbon/human/vv_get_dropdown()
-	. = ..()
-	. += "---"
-	.["Set Species"] = "?_src_=vars;setspecies=[UID()]"
-	.["Copy Outfit"] = "?_src_=vars;copyoutfit=[UID()]"
-	.["Make AI"] = "?_src_=vars;makeai=[UID()]"
-	.["Make cyborg"] = "?_src_=vars;makerobot=[UID()]"
-	.["Make monkey"] = "?_src_=vars;makemonkey=[UID()]"
-	.["Make alien"] = "?_src_=vars;makealien=[UID()]"
-	.["Make slime"] = "?_src_=vars;makeslime=[UID()]"
-	.["Make superhero"] = "?_src_=vars;makesuper=[UID()]"
-	. += "---"
-
```

Comment:
Remove the callbacks for these maybe?

Reply:
they are used in PP

Reply:
> they are used in PP

Not exactly.
![image](https://user-images.githubusercontent.com/25063394/187790465-6a27a793-ad56-4a31-b932-6d972b00a36a.png)

Reply:
Oh they are in a different topic? Well damn, alright I'll see what I can do